src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -361,6 +368,75 @@ def _get_hypervisor(host_name, server_id):
             ))
             return None
 
+    def _match_physical_and_cloud_hosts(self):
+        """Connect CloudHosts and DC assets according to data from Ironic."""
+
+        provider = getattr(
+            settings, 'OPENSTACK_IRONIC_PROVIDER', 'openstack-ironic'
+        )
+        os_conf = next((conf for conf in settings.OPENSTACK_INSTANCES
```

Comment:
here you're processing only first found ironic config - what if there are multiple of them?